lib/widgets/custom_switch_tile.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+class CustomSwitchTile extends StatelessWidget {
```

Comment:
Looks a bit small for an own widget. Is it essential?

Reply:
I created it to centralize the UI(how theme and TextStyle) of all SwitchTile.